### PR TITLE
Add load spinner to the transcript, lecture list, and comments.

### PR DESCRIPTION
**Pull request by amuamushu**
The spinners are in the transcript, comments, and lecture list containers by default. Once the items are loaded, they are removed from their respective containers.

Spinners in the transcript and comment sections: https://screenshot.googleplex.com/5qUBnFPT49fVazi.png
Spinner in the lecture list: https://screenshot.googleplex.com/3y4gM3tTPtpy767.png

### CODE REVIEW

File: src/main/webapp/view/lecture-view.css [hunk 1]
```diff
@@ -128,6 +132,10 @@ discussion-comment[highlighted] > [slot='header'] {
   padding: 15px;
 }
 
+#transcript-container .spinner-border {
+  margin-left: 40%;
```
Comment (n-wach) [inline src/main/webapp/view/lecture-view.css:136]: nit: check if margin auto works to center horizontally here

File: src/main/webapp/view/lecture-view.css [hunk 2]
```diff
@@ -54,6 +54,10 @@ body {
   padding: 15px;
 }
 
+#discussion .spinner-border {
+  margin-left: 45%;
```
Comment (n-wach) [inline src/main/webapp/view/lecture-view.css:58]: ditto here

Comment (amuamushu) [reply src/main/webapp/view/lecture-view.css:58]: (resolved in chat) margin-auto unfortunately doesn't work :(

File: src/main/webapp/view/lecture-view.css (hunk 1 above)
Comment (amuamushu) [reply src/main/webapp/view/lecture-view.css:136]: (resolved in chat) margin-auto unfortunately doesn't work :(

**Pull request merged**